web/profiles/custom/os2loop/modules/os2loop_messages/src/Helper/Helper.php

```diff
@@ -48,7 +62,13 @@ private function createMessage(EntityInterface $entity, string $operation) {
     if (NULL !== $template) {
       $message = Message::create(['template' => $template]);
       if ($entity instanceof NodeInterface) {
+        if ($entity->hasField('os2loop_notify_users') && FALSE == $entity->get('os2loop_notify_users')->getValue()[0]['value']) {
```

Comment:
The expression `FALSE == $entity->get('os2loop_notify_users')->getValue()[0]['value']` makes a lot of assumptions on what `getValue()` returns.
I'm not sure what *the right way* to do it is, but using `$entity->get('os2loop_notify_users')->getString()` may be useful.

Reply:
CHanged